src/app/client/src/app/modules/shared-feature/components/program-list/program-list.component.ts

```diff
@@ -230,8 +185,10 @@ export class ProgramListComponent implements OnInit, AfterViewInit {
 
   getProgramsListByRole(setfilters?) {
     // debugger;
-      console.log(this.cslFrameworkService?.getFrameworkCategories());
-      console.log(this.cslFrameworkService?.getFrameworkCategoriesObject());
+      //console.log(this.cslFrameworkService?.getFrameworkCategories());
+    this.frameworkObjectFields = this.cslFrameworkService?.getFrameworkCategoriesObject();
+    this.frameworkFields = _.map(this.frameworkObjectFields, 'label').slice(0,3).join(',');
```

Comment:
What is this slice(0,3).. 

if framework is handling only 2 categories then what?

Why are we writing these kind of logics without discussions..

Reply:
If the fields are more than 3 or 4 the UI will break and use will not see the open button for the project. I wanted to show the same to you before raising this PR.